### PR TITLE
Fix additional url imports

### PR DESCRIPTION
**Pull request by DavisRayM**
# Changes

- [x] Fix url imports

### CODE REVIEW

**Review by ivermac — CHANGES_REQUESTED** [review]
Looks good to me. Could you please add a test for `OpenIDConnectViewset `

**Comment by ukanga** [comment]
We need to address this as a python django app that is sideloaded with RapidPro, we should perhaps only change the local settings.py.

File: temba/urls.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
 # import any additional urls
 for app in settings.APP_URLS:  # pragma: needs cover
-    importlib.import_module(app)
+    urlpatterns.append(url(r"^", include(app)))
```
Comment (DavisRayM) [inline temba/urls.py:53]: This didn't seem like it was registering the urls... seems like it's just importing the module ?

File: templates/includes/nav.haml [hunk 2]
```diff
@@ -259,6 +259,12 @@
       .title
         -trans "sign in"
 
+    %a{href:'{% url "oidc:openid_connect_login" auth_server="msft" %}'}
+      .circle
+        .icon-user
+      .title
+        -trans "sign in with microsoft"
```
Comment (ivermac) [inline templates/includes/nav.haml]: This is for microsoft's OpenID connect login alone ama?

Comment (DavisRayM) [reply templates/includes/nav.haml]: Yes, it is.

Comment (DavisRayM) [reply templates/includes/nav.haml]: _This is just a temporary change though. For a deployment we can override this template and change the url for the main 'Sign in' button_

**Review by ivermac — APPROVED** [review]

File: temba/urls.py (hunk 1 above)
Comment (ukanga) [reply temba/urls.py:53]: Should we be configuring our `APP_URLS` as below?
```python
APP_URLS = ['mycustomapp.urls', ]
```
 An import of the URLs module should register URLs, you do not need to explicitly do `urlpatterns.append`. Did you test this and it was not working?

Comment (DavisRayM) [reply temba/urls.py:53]: Yes, I tested this and it wasn't picking up the urls. Might it be an issue with how the urls are declared in `ona-oidc` ?

**Pull request closed without merging**